src/admm/acopf_admm_gpu.jl

```diff
@@ -1,14 +1,26 @@
-function get_generator_data(data::OPFData; use_gpu=false)
+function get_generator_data(data::OPFData; device = KA.CPU())
     ngen = length(data.generators)
 
-    if use_gpu
-        pgmin = CuArray{Float64}(undef, ngen)
-        pgmax = CuArray{Float64}(undef, ngen)
-        qgmin = CuArray{Float64}(undef, ngen)
-        qgmax = CuArray{Float64}(undef, ngen)
-        c2 = CuArray{Float64}(undef, ngen)
-        c1 = CuArray{Float64}(undef, ngen)
-        c0 = CuArray{Float64}(undef, ngen)
+    if isa(device, KA.GPU)
+        if isa(device, CUDADevice)
```

Comment:
I will directly write the `if` on `CUDADevice` and `ROCDevice` (and skip the `KA.GPU` condition)